UserDetailsClient/UserDetailsClient.Droid/Resources/Resource.Designer.cs

```diff
@@ -15,7 +15,7 @@ namespace UserDetailsClient.Droid
 {
 	
 	
-	[System.CodeDom.Compiler.GeneratedCodeAttribute("Xamarin.Android.Build.Tasks", "1.0.0.0")]
+	[global::System.CodeDom.Compiler.GeneratedCodeAttribute("Xamarin.Android.Build.Tasks", "1.0.0.0")]
```

Comment:
this is an automatic change that should be undone

Reply:
done